logger.js

```diff
@@ -0,0 +1,3 @@
+import createPino from "pino";
```

Comment:
Since eslint complains if you use `console.log`, here I set up logger with pino (which is a lightweight logging library).